src/Azure/AzureAD/test/FunctionalTests/Microsoft.AspNetCore.Authentication.AzureAD.FunctionalTests.csproj

```diff
@@ -23,18 +26,21 @@
     </ItemGroup>
     <Copy
       SourceFiles="@(_PublishFiles)"
-      DestinationFolder="$(PublishDir)" />
+      DestinationFolder="$(PublishDir)"
+      ContinueOnError="true" />
```

Comment:
This looks odd. Won't the tests just waste time and fail later if assets are incomplete? Seems better to fail fast.

Reply:
This error is actually typically a double copy failure